src/Xablu.WebApiClient/Xablu.WebApiClient.csproj

```diff
@@ -92,9 +93,7 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
No it won't it only removes the explicit folder declaration. The folder will be implicitly inferred from the file declaration on line 52: `<Compile Include="HttpContentExtensions\ProgressStreamContent.cs" />`

So the file and even the folder will still be there.